tee-worker/app-libs/stf/src/trusted_call.rs

```diff
@@ -579,6 +584,21 @@ where
 					Ok(TrustedCallResult::Streamed)
 				}
 			},
+
+			TrustedCall::remove_identity(signer, who, identity) => {
```

Comment:
I think we shall add the production guard here, like error out for prod directly.

Reply:
Make use of `if_production_or`. Maybe simply give some message out that it's not supported in production mode.

Reply:
Sounds good 👍